lib/event.rb

```diff
@@ -0,0 +1,67 @@
+ages = [24,30,18,20,41]
+
+class Event
+  attr_reader :name,
+              :ages
+  def initialize(name,ages=[])
+    @name = name
+    @ages = ages
+  end
+
+  def max_age
+    @ages.max
+  end
+
+  def min_age
+    @ages.min
+  end
+
+  def average_age
+    average = @ages.sum.to_f / @ages.length
+    average.round(2)
+  end
+
+  def standard_deviation_age
+    input = @ages
+    sum = get_sum(input)
+    average = get_average(sum)
```

Comment:
Instead of these two lines, you can reuse your average_age method. It would look like this:

```ruby
average = average_age
```